release-notes/3.0/preview/api-diff/preview4/3.0-preview4_System.Runtime.Intrinsics.X86.md

```diff
@@ -0,0 +1,83 @@
+# System.Runtime.Intrinsics.X86
+
+``` diff
+ namespace System.Runtime.Intrinsics.X86 {
+     public abstract class Avx : Sse42 {
+-        public unsafe static void ExtractVector128(byte* address, Vector256<byte> value, byte index);
```

Comment:
Another note: These were removed since we implicitly emit the memory store encoding when necessary.

Likewise for `InsertVector128` we implicitly emit the memory load encoding when necessary.